macosx/GroupTextCell.mm

```diff
@@ -22,9 +23,15 @@ - (void)drawInteriorWithFrame:(NSRect)cellFrame inView:(NSView*)controlView
     //set font size and color
     NSRect titleRect = [self titleRectForBounds:cellFrame];
     NSMutableAttributedString* string = [[self attributedStringValue] mutableCopy];
-    NSDictionary* attributes = [NSDictionary
+    NSMutableDictionary* attributes = [NSMutableDictionary
         dictionaryWithObjects:@[ [NSFont boldSystemFontOfSize:11.0], [NSColor secondaryLabelColor] ]
                       forKeys:@[ NSFontAttributeName, NSForegroundColorAttributeName ]];
+
+    if (self.selected)
+    {
+        [attributes setObject:[NSColor labelColor] forKey:NSForegroundColorAttributeName];
```

Comment:
nit: modern obj-c could be used.

Reply:
You're actually setting NSForegroundColorAttributeName twice, so I'd prefer to  go for a ternary pattern or an `if {} else {}` pattern.

Example:
```
NSDictionary* attributes = [NSDictionary
         dictionaryWithObjects:@[ [NSFont boldSystemFontOfSize:11.0], selected ? [NSColor labelColor] : [NSColor secondaryLabelColor] ]
                       forKeys:@[ NSFontAttributeName, NSForegroundColorAttributeName ]];
```